src/core/server/plugins/plugins_service.ts

```diff
@@ -206,7 +206,7 @@ export class PluginsService implements CoreService<PluginsServiceSetup, PluginsS
     >();
     await plugin$
       .pipe(
-        mergeMap(async (plugin) => {
+        concatMap(async (plugin) => {
```

Comment:
We got unit tests asserting that plugins received from `discover` so it seems to be properly working already, but just in case, I switched from `mergeMap` to `concatMap`